dags/copy_deduplicate.py

```diff
@@ -110,6 +110,7 @@
             task_id="graphics_telemetry__wait_for_main_ping",
             external_dag_id="graphics_telemetry",
             external_task_id="wait_for_main_ping",
+            execution_date="{{ (execution_date + macros.timedelta(hours=2)).isoformat() }}",
```

Comment:
When we implement this more broadly I'd suggest hard-coding the time part rather than having it be an offset from the upstream DAG's schedule, because this will break if either the upstream DAG's schedule or downstream DAG's schedule changes.

With the external task execution time hard-coded it will only break if the downstream DAG's schedule changes, and it should be easier to implement (just read the downstream schedule rather than having to calculate the delta to use).

Reply:
Good point!